repl/scala-2.12/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -64,10 +61,10 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
           _sc
         }
         """)
-      processLine("import org.apache.spark.SparkContext._")
-      processLine("import spark.implicits._")
-      processLine("import spark.sql")
-      processLine("import org.apache.spark.sql.functions._")
+      command("import org.apache.spark.SparkContext._")
```

Comment:
This was the real change. Also part of the reason Spark didn't work in 2.11.11. This avoids waiting on `ILoop.globalFuture` which isn't initialized when processLine gets called anymore